src/patterns/data/default/entity_attribute_location.tsv

```diff
@@ -248,5 +248,5 @@ OBA:2020005	lysine in blood amount	CHEBI:25094	lysine	PATO:0000070	amount	UBERON
 OBA:2060174		CHEBI:145810	insulin	PATO:0000070	amount	UBERON:0000178	blood
 OBA:2060175		CHEBI:35143	hemoglobin	PATO:0000070	amount	UBERON:0000178	blood
 OBA:2060176		CHEBI:25107	magnesium atom	PATO:0000070	amount	UBERON:0001088	urine
-OBA:2060177		PR:000022172	aspartate aminotransferase	PATO:0000070	amount	UBERON:0000178	blood
+OBA:2060177             PR:000022172    aspartate aminotransferase      PATO:0000070    amount  UBERON:0000178  blood
```

Comment:
Intended change?

Reply:
I had removed it, but after having second thoughts, I have restored this term.
My problem was that `PR:000022172 ` is "A protein that is a translation product of the Escherichia coli K-12 aspC gene or a 1:1 ortholog thereof". Thus it is a foreign protein which is not normally a part of or found in blood. However, I think it sill can be considered a trait, because the new term is independent of the state of the character (i.e. amount = 0 --> normal; amount > 0 abnormal).